client/src/components/Portfolio.jsx

```diff
@@ -51,19 +51,35 @@ class Portfolio extends Component {
                 <th> Date changed </th>
               </tr>
             </thead>
-            <tbody>
-            {items.map((it, i) => (
-              <tr key={i} onClick={() => this.onSelectPortfolioItem({ ticker: it.symbol, companyName: it.company_name })} style={{ cursor: 'pointer' }}>
-                <td> {it.symbol} </td>
-                <td>
-                  {it.symbol === 'USD' ? '-' : this.formatQuantity(it.quantity, it.symbol)}
-                </td>
-                <td> {it.symbol === "USD" ? "-": `\$${this.formatQuantity(it.price_per_share, 'USD')}`} </td>
-                <td> ${it.symbol === "USD" ? this.formatQuantity(it.quantity, 'USD') : this.getStockValueNumber(it.quantity, it.price_per_share)}</td>
-                <td> {timeago.format(it.date_changed)} </td>
-              </tr>
-            ))}
+            {enableClick ? (
+              <tbody>
```

Comment:
there's a cleaner way to do this that doesn't duplicate the  JSX code. We can just check `enableClick` in `onSelectPortfolioItem`, and have it return immediately when it is false.